configuration/policy.yml

```diff
@@ -0,0 +1,66 @@
+server:
+  applicationConnectors:
+    - type: http
+      port: 8080
+  requestLog:
+    type: classic
+    appenders:
+      - type: access-logstash-console
+timeoutPeriod: 90m
+assertionLifetime: 60m
+acceptSelfSignedCerts: true
+matchingServiceResponseWaitPeriod: 60s
+sessionStore:
+  redis:
+    recordTTL: PT150m
+    uri: ${REDIS_HOST}
+samlEngineUri: https://saml-engine.${DOMAIN}:443
```

Comment:
We could omit the explicit port 443 on HTTPS URLs, presumably

Reply:
(this is not worth holding this PR up, just jotting down comments as I'm looking through the diff)

Reply:
I agree with @rhowe-gds. 443 is the default port number for HTTPS. It is safe to remove the explicit port number.

Reply:
Actually this is already the case for saml-soap-proxy, for example
```
samlEngineUri: https://saml-engine.${DOMAIN}
configUri: https://config.${DOMAIN}
policyUri: https://policy.${DOMAIN}
```